Stratis.Bitcoin.Tests/BlockStore/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Stratis.Bitcoin.Tests.BlockStore.LoopTests
 {
+    /// <summary>
+    /// Base test class for all the BlockStoreLoop tests
+    /// </summary>
     public class BlockStoreLoopStepBaseTest
```

Comment:
Fixed the <see> comment. Yes, I'm not sure whether or not the unit tests needs to commented extensively like that. I have added inline comments where applicable and the name of the unit test should be enough of an indication as what is being tested as well as the potential outcome.